lib/handler/status/durations.c

```diff
@@ -125,12 +134,23 @@ static h2o_iovec_t durations_status_final(void *priv, h2o_globalconf_t *gconf, h
             "request-total-time") "," DURATION_FMT("process-time") "," DURATION_FMT("response-time") "," DURATION_FMT("duration"),
         DURATION_VALS(connect_time), DURATION_VALS(header_time), DURATION_VALS(body_time), DURATION_VALS(request_total_time),
         DURATION_VALS(process_time), DURATION_VALS(response_time), DURATION_VALS(total_time));
-
-#undef BUFSIZE
 #undef DURATION_FMT
 #undef DURATION_VALS
-
+    char *delim = "";
+    ret.len += sprintf(ret.base + ret.len, ",\n\"evloop-latency-nanosec\": [");
+    for(int i = 0; i < agg_stats->stats.evloop_latency_nanosec.size; i++) {
+        size_t len = snprintf(NULL, 0, "%s%llu", delim, agg_stats->stats.evloop_latency_nanosec.entries[i]);
+        /* require that there's enough space for the closing array bracket */
+        if (ret.len + len + 1 > BUFSIZE)
```

Comment:
Shouldn't the rational operator be `>=`?

We need space for `]` and `\0` after the text being emitted in this for block.

Reply:
That's correct, thanks for catching this.